src/nvim/testdir/test_arglist.vim

```diff
@@ -1,5 +1,10 @@
 " Test argument list commands
 
+func Reset_arglist()
+  cd
```

Comment:
https://github.com/neovim/neovim/pull/14540#discussion_r636577072
Same comment about `cd` for Unix systems.